kokoro/linux/bazel/presubmit.cfg

```diff
@@ -0,0 +1,5 @@
+# Config file for running tests in Kokoro
+
+# Location of the build script in repository
+build_file: "protobuf/kokoro/linux/bazel/build.sh"
+timeout_mins: 1440
```

Comment:
Have this test run before? 1440 seems to be too long, I think if some PR does something wrong that makes some test deadlock forever then the presubmit will block following PR's presubmit.

Reply:
Changed to 15 instead.